web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -678,6 +678,17 @@
       $scope.session = gnSessionService.getSession();
 
       $scope.loadCatalogInfo();
+
+
+      $scope.healthCheck = {};
```

Comment:
Is this not redundant with the healthchecks in the admin status page? Haven't checked but I was expecting some code removal along this addition.

Reply:
It does a bit more with critical, warning https://github.com/geonetwork/core-geonetwork/blob/master/web-ui/src/main/resources/catalog/js/admin/DashboardStatusController.js#L127-L141